src/components/flat-table/flat-table-row/flat-table-row.style.js

```diff
@@ -114,9 +114,13 @@ const StyledFlatTableRow = styled.tr`
 
       ${StyledFlatTableHeader} {
         border-bottom: 1px solid ${borderColor(colorTheme, theme)};
-        :first-child {
-          border-left: 1px solid ${borderColor(colorTheme, theme)};
-        }
+
+        ${!isInSidebar &&
+        `
```

Comment:
```suggestion
        css`
```

Reply:
`css` is used at the `return` so I don't think it's needed here as well?